src/pages/index.js

```diff
@@ -64,46 +54,37 @@ export const getServerSideProps = async () => {
 };
 
 export default function Home(getServerSideProps) {
```

Comment:
Please destructure `newsAll` and `profileAll` instead of passing `getServerSideProps` directly: `{ newsAll, profileAll }`

Reply:
I am not sure what you are asking for.

Reply:
Oh, I see now. I think I know what you mean.